### PR TITLE
7.8/search

### PR DESCRIPTION
**Pull request by lguardado**
#### :page_facing_up: Description:

Add search functionality and test coverage

---

#### :pushpin: Notes:

--

---

#### :heavy_check_mark: Performed Tasks:

- Added search
- Using debounce call to API
- Added tests

---

#### :play_or_pause_button: Demo:
iOS
Kapture 2020-12-15 at 11 28 31
Android
Kapture 2020-12-15 at 12 25 57

### CODE REVIEW

**Review by Jmaguerre — COMMENTED** [review]
Looks good, just one comment

File: src/components/Search/Search.js [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+      setErrorMsg(error.message || strings.errorFetching);
+    }
+    setIsLoading(false);
+  };
+
+  const handleChangeText = value => {
+    setIsLoading(true);
+    setQuery(value);
+    const delay = value.length ? 500 : 0;
+    const search = _.debounce(async term => performQuery(term), delay);
```
[the remark this answers is not in the document]
Comment (lguardado) [reply src/components/Search/Search.js:55]: Actually I tried that and it wasn't working, after that I realised I was missing a return statement and I didn't go back to that solution. Let me try again and I'll get back to you

[the remark this answers is not in the document]
Comment (lguardado) [reply src/components/Search/Search.js]: Done @Jmaguerre! thanks for the advice. I like this hook much more than the previous solution

**Review by Jmaguerre — APPROVED** [review]
